include/qdmi/device.h

```diff
@@ -593,6 +642,250 @@ void QDMI_device_job_free(QDMI_Device_Job job);
 
 /** @} */ // end of device_job_interface
 
+/** @defgroup device_telemetrysensor_query_interface \
+ * QDMI Device Telemetry Sensor Query Interface
+ *  @brief Provides functions to manage telemetry sensor queries on a device.
+ *  @details An telemetry sensor query is a task submitted to a device for
```

Comment:
```suggestion
 *  @details A telemetry sensor query is a task submitted to a device for
```
I may have missed some occurrences of "An tele...". Could you please check whether there are more?